frontend/src/tabs/authorship.pug

```diff
@@ -1,8 +1,8 @@
 #authorship
   span.large-font Code Panel
   .toolbar(v-if="hasCommits(info)")
-    a(v-if="activeFilesCount === 0", v-on:click="expandAll(true)") show all file details
-    a(v-else, v-on:click="expandAll(false)") hide all file details
+    a(v-if="activeFilesCount < this.selectedFiles.length", v-on:click="expandAll()") show all file details
+    a(v-else, v-on:click="collapseAll()") hide all file details
```

Comment:
Since we are using vue, is it better we just watch the activeFilesCount instead of implementing 2 click functions? Also, why revamp the previous logic for button change? I feel display `hide all` as long as there is at least one active file would be better since the first thing I do when open the authorship view is usually to hide all file details.

Reply:
> Since we are using vue, is it better we just watch the activeFilesCount instead of implementing 2 click functions?

Hmm I don't follow, isn't vue already watching `activeFilesCount` implicitly and showing/hiding the `a(...)`'s as needed with the `v-if`'s?


> Also, why revamp the previous logic for button change? I feel display hide all as long as there is at least one active file would be better since the first thing I do when open the authorship view is usually to hide all file details.

Fixed :+1:. Only meant to change the condition for the first `a(...)`